libs/orders/src/lib/order-hooks/use-order-validation.tsx

```diff
@@ -186,14 +128,17 @@ export const useOrderValidation = ({
       };
     }
 
-    if (fieldErrors?.price?.type === 'required') {
+    if (
+      fieldErrors?.price?.type === 'required' &&
```

Comment:
chore (blocking): You shouldn't have to conditionally change the error message by `type` if using the validate property in the `register` function of the field.

Reply:
the reason why tried to handle most the of the warning messages in 'useOrderValidation' hook is because I wanted to handle better the order in which these messages appear and to do that in one place. 
 
 i.e. here,  if I move these rules to validate, works well if the market is trading continuously but if it is in a monitoring auction then the 'You cannot set a negative price` error gets overridden by `Any orders traded now will not trade until the execution ends`
 
 
![image](https://user-images.githubusercontent.com/16125548/185107439-c6995691-a42a-45aa-94b5-5fd519e69c0d.png)

Reply:
I can handle this only in register, if I add the market trading mode as a field in deal ticket limit amount where we register the form and move more rules over there, but is it better?

Reply:
No I think this is fine. Its out side the component itself so I think here its better to use the fieldErrors object.